OpenEdXMobile/src/main/java/org/edx/mobile/module/registration/view/RegistrationEditTextView.java

```diff
@@ -15,79 +17,67 @@
 import org.edx.mobile.logger.Logger;
 import org.edx.mobile.module.registration.model.RegistrationFormField;
 
-import java.util.Locale;
-
 class RegistrationEditTextView implements IRegistrationFieldView {
 
     protected static final Logger logger = new Logger(RegistrationEditTextView.class);
     protected RegistrationFormField mField;
     private View mView;
-    protected EditText mInputView;
-    private TextView mErrorView, mInstructionView;
-    private IActionListener actionListener;
+    protected TextInputLayout mTextInputLayout;
+    protected TextInputEditText mTextInputEditText;
+    protected TextView mInstructionsTextView;
+    protected TextView mErrorTextView;
```

Comment:
They are references by subclasses. They're protected in  master too (and are accessed directly)